src/modules/admin/AdminPage.vue

```diff
@@ -45,12 +45,14 @@
       </div>
     </div>
     <div class="flex-1 basis-2/3">
-      <template v-if="latestCallout">
-        <AppHeading>{{ t('adminDashboard.latestCallout.title') }}</AppHeading>
-        <a class="block bg-white p-4 mt-4 mb-8 rounded">
-          <CalloutSummary :callout="latestCallout" footer />
-        </a>
-      </template>
+      <AppHeading>{{ t('adminDashboard.latestCallout.title') }}</AppHeading>
+      <p v-if="latestCallout" class="block bg-white p-4 mt-4 mb-8 rounded">
```

Comment:
Looks great but I think the `v-if` and `v-else-if` might be the wrong way round :stuck_out_tongue:

Reply:
D'oh, of course. All fixed, thank you for the careful review :stars: